lib/libc/minimal/include/limits.h

```diff
@@ -80,8 +80,129 @@ extern "C" {
 #error "unexpected __SIZEOF_LONG_LONG__ value"
 #endif
 
+#if defined(_POSIX_C_SOURCE) || defined(__DOXYGEN__)
+
+/*
+ * clang-format and checkpatch disagree on formatting here, so use rely and checkpatch and disable
```

Comment:
Copy / paste typo, probably should be corrected.
```suggestion
 * clang-format and checkpatch disagree on formatting here, so use rely and checkpatch and disable
```